tools/update_tool

```diff
@@ -0,0 +1,466 @@
+
+RESET = "\033[0m"
+GREEN = "\033[32m"
+BOLD = "\033[1m"
+RED = "\033[31m"
+YELLOW = "\033[33m"
+STRIKE = "\033[9m"
+
+# Utility functions to print an ascii table and a github markdown lits
+
+def start_row(text, gh):
+    end = "|"
+    prefix = ""
+    l = 77
+    if gh:
+        l = 0
+        end = ", "
+        prefix = "- [ ] "
+
+    print(f"{prefix}{text}".ljust(l), end=end)
+
+
+def green_adjusted(text, bold=False, strike=False, l=18, gh=False, last=False):
+    bo = st = end = ""
+    if gh:
+        l = 0
+        end = ", "
+        if last:
+            end = "."
+        if bold:
+            bo = "**"
+        if strike:
+            st = "*"
+        print(f"{bo}{st}{text}{st}{bo}".ljust(l), end=end)
+    else:
+        end = "|"
+        if bold:
+            bo = BOLD
+        st = ""
+        if strike:
+            st = STRIKE
+        print(f"{GREEN}{bo}{st}{text.ljust(l)}{RESET}", end=end)
+
+
+def red_adjusted(text, bold=False, strike=True, l=18, gh=False, last=False):
+    bo = st = end = ""
+    if gh:
+        l = 0
+        end = ", "
+        if last:
+            end = "."
+        if bold:
+            bo = "**"
+        if strike:
+            st = "*"
+        print(f"{bo}{st}{text}{st}{bo}".ljust(l), end=end)
+    else:
+        end = "|"
+        if bold:
+            bo = BOLD
+        st = ""
+        if strike:
+            st = STRIKE
+        print(f"{RED}{bo}{st}{text.ljust(l)}{RESET}", end=end)
+
+
+def print_header(gh):
+    end = "|"
+    end1 = "|"
+    prefix = ""
+    l = 77
+    l1 = 18
+    if gh:
+        l = 0
+        l1 = 0
+        end = ", "
+        end1 = "."
+        prefix = "Please review each "
+    print(f"{prefix}DB entry".ljust(l), end=end)
+    print("manifest".ljust(l1), end=end)
+    print("image-info".ljust(l1), end=end1)
+    print()
+    if not gh:
+        print("-"*116)
+
+def print_status(status, gh, last=False):
+    if status == "added":
+        if gh:
+            green_adjusted("🆕 added", bold=True, gh=True, last=last)
+        else:
+            green_adjusted("added", bold=True, last=last)
+    elif status == "updated":
+        if gh:
+            green_adjusted("🔁 updated", gh=True, last=last)
+        else:
+            green_adjusted("updated", last=last)
+    elif status == "updated":
+        if gh:
+            red_adjusted("🗑️ removed", strike=True, gh=True, last=last)
+        else:
+            red_adjusted("removed", strike=True, last=last)
+    else:
+        if gh:
+            red_adjusted("💾 unchanged", gh=True, last=last)
+        else:
+            red_adjusted("unchanged", last=last)
+
+def end_row(gh):
+    print()
+    if not gh:
+        print("-"*116)
+
+
+def compare(obj1, obj2):
+    """
+    Compare two json objects and return as a string the conclusion of the diff.
+    Obj1 is the version of Obj2 back in time. So when obj2 is not and obj1 is,
+    we can say that something was removed and vice versa.
+    """
+    if obj1 and not obj2:
```

Comment:
Bit of a nitpick, but is an empty `obj1` (say: `{}`) the same as `obj1` being `None`? If it isn't it needs an identity check for the latter in these conditionals.

Reply:
That's a good one! Thanks for this. I'd say, they should be considered different, something changed in the DB, so, we should highlight that.